code/game/objects/structures/tables_racks.dm

```diff
@@ -136,18 +136,22 @@
 	log_combat(user, pushed_mob, "tabled", null, "onto [src]")
 	SEND_SIGNAL(pushed_mob, COMSIG_ADD_MOOD_EVENT, "table", /datum/mood_event/table)
 
-/obj/structure/table/proc/tableheadsmash(mob/living/user, mob/living/pushed_mob)
+/obj/structure/table/proc/tablelimbsmash(mob/living/user, mob/living/pushed_mob)
 	pushed_mob.Knockdown(30)
-	pushed_mob.apply_damage(30, BRUTE, BODY_ZONE_HEAD)
-	pushed_mob.apply_damage(40, STAMINA)
+	var/obj/item/bodypart/banged_limb = pushed_mob.get_bodypart(user.zone_selected) || pushed_mob.get_bodypart(BODY_ZONE_HEAD)
+	var/extra_wound = 0
+	if(HAS_TRAIT(user, TRAIT_HULK))
+		extra_wound = 20
+	banged_limb.receive_damage(40, wound_bonus = extra_wound)
+	pushed_mob.apply_damage(60, STAMINA)

```

Comment:
I see you tryin' ta sneak a tabling buff through here

Reply:
lowered damage again